packages/core/patternlab-config.js

```diff
@@ -0,0 +1,122 @@
+const resolvePkg = require('resolve-pkg');
+
+module.exports = {
+  engines: {
+    twig: {
+      namespaces: [
+        {
+          id: 'uikit',
+          recursive: true,
+          paths: [resolvePkg(`@pattern-lab/uikit-workshop/views-twig`)],
```

Comment:
I don't think we need `resolvePkg()`; [`require.resolve()` is in core](https://nodejs.org/api/modules.html#modules_require_resolve_request_options) and returns the absolute path instead of loading it. One less dependency! 🎉

Reply:
OK, I see the need for the package now; we need `resolvePkg()` to be able to resolve folders, which `require.resolve` can't do.